open-metadata-implementation/adapters/open-connectors/repository-services-connectors/open-metadata-collection-store-connectors/graph-repository-connector/src/main/java/org/odpi/openmetadata/adapters/repositoryservices/graphrepository/repositoryconnector/GraphOMRSMetadataCollection.java

```diff
@@ -321,7 +321,7 @@ public EntityDetail isEntityKnown(String     userId,
             repositoryValidator.validateEntityFromStore(repositoryName, guid, entity, methodName);
         }
         catch (EntityProxyOnlyException | EntityNotKnownException e) {
-            log.error("{} entity with GUID {} does not exist in repository {} or is a proxy", methodName, guid, repositoryName);
+            log.warn("{} entity with GUID {} does not exist in repository {} or is a proxy", methodName, guid, repositoryName);
```

Comment:
This is odd logic, the caller is a method that is adding a proxy, if it finds a proxy here then null is returned and it creates a proxy!